File: src/main/webapp/app/entities/config-variables/config-variables.test-samples.ts
import { IConfigVariables, NewConfigVariables } from './config-variables.model';

export const sampleWithRequiredData: IConfigVariables = {
  id: 22518,
};

export const sampleWithPartialData: IConfigVariables = {
  id: 5502,
  configVarLong1: 2499,
  configVarLong2: 17356,
  configVarLong3: 7053,
  configVarLong4: 13768,
  configVarLong5: 5983,
  configVarLong6: 12318,
  configVarLong7: 18997,
  configVarLong12: 20024,
  configVarLong13: 19339,
  configVarLong14: 28528,
  configVarLong15: 18832,
  configVarBoolean17: true,
  configVarString19: 'dearly reprimand',
  configVarString20: 'even kindheartedly',
};

export const sampleWithFullData: IConfigVariables = {
  id: 25730,
  configVarLong1: 32679,
  configVarLong2: 4406,
  configVarLong3: 23682,
  configVarLong4: 3590,
  configVarLong5: 554,
  configVarLong6: 28664,
  configVarLong7: 20754,
  configVarLong8: 13159,
  configVarLong9: 1972,
  configVarLong10: 25064,
  configVarLong11: 16225,
  configVarLong12: 24903,
  configVarLong13: 26259,
  configVarLong14: 10671,
  configVarLong15: 32763,
  configVarBoolean16: true,
  configVarBoolean17: true,
  configVarBoolean18: true,
  configVarString19: 'decryption righteously apud',
  configVarString20: 'the decouple comfortable',
};

export const sampleWithNewData: NewConfigVariables = {
  id: null,
};

Object.freeze(sampleWithNewData);
Object.freeze(sampleWithRequiredData);
Object.freeze(sampleWithPartialData);
Object.freeze(sampleWithFullData);

File: src/test/java/com/opencode/test/domain/ConfigVariablesTestSamples.java
package com.opencode.test.domain;

import java.util.Random;
import java.util.UUID;
import java.util.concurrent.atomic.AtomicLong;

public class ConfigVariablesTestSamples {

    private static final Random random = new Random();
    private static final AtomicLong longCount = new AtomicLong(random.nextInt() + (2L * Integer.MAX_VALUE));

    public static ConfigVariables getConfigVariablesSample1() {
        return new ConfigVariables()
            .id(1L)
            .configVarLong1(1L)
            .configVarLong2(1L)
            .configVarLong3(1L)
            .configVarLong4(1L)
            .configVarLong5(1L)
            .configVarLong6(1L)
            .configVarLong7(1L)
            .configVarLong8(1L)
            .configVarLong9(1L)
            .configVarLong10(1L)
            .configVarLong11(1L)
            .configVarLong12(1L)
            .configVarLong13(1L)
            .configVarLong14(1L)
            .configVarLong15(1L)
            .configVarString19("configVarString191")
            .configVarString20("configVarString201");
    }

    public static ConfigVariables getConfigVariablesSample2() {
        return new ConfigVariables()
            .id(2L)
            .configVarLong1(2L)
            .configVarLong2(2L)
            .configVarLong3(2L)
            .configVarLong4(2L)
            .configVarLong5(2L)
            .configVarLong6(2L)
            .configVarLong7(2L)
            .configVarLong8(2L)
            .configVarLong9(2L)
            .configVarLong10(2L)
            .configVarLong11(2L)
            .configVarLong12(2L)
            .configVarLong13(2L)
            .configVarLong14(2L)
            .configVarLong15(2L)
            .configVarString19("configVarString192")
            .configVarString20("configVarString202");
    }

    public static ConfigVariables getConfigVariablesRandomSampleGenerator() {
        return new ConfigVariables()
            .id(longCount.incrementAndGet())
            .configVarLong1(longCount.incrementAndGet())
            .configVarLong2(longCount.incrementAndGet())
            .configVarLong3(longCount.incrementAndGet())
            .configVarLong4(longCount.incrementAndGet())
            .configVarLong5(longCount.incrementAndGet())
            .configVarLong6(longCount.incrementAndGet())
            .configVarLong7(longCount.incrementAndGet())
            .configVarLong8(longCount.incrementAndGet())
            .configVarLong9(longCount.incrementAndGet())
            .configVarLong10(longCount.incrementAndGet())
            .configVarLong11(longCount.incrementAndGet())
            .configVarLong12(longCount.incrementAndGet())
            .configVarLong13(longCount.incrementAndGet())
            .configVarLong14(longCount.incrementAndGet())
            .configVarLong15(longCount.incrementAndGet())
            .configVarString19(UUID.randomUUID().toString())
            .configVarString20(UUID.randomUUID().toString());
    }
}
